lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -19,25 +19,48 @@
   }
 
   @mixin accordion-title() {
-    @include side-nav-title();
+    @include font-size(0);
+    border: 1px solid $light-gray;
+    margin: -1px 0 0 0;
+    color: $dark-gray;
+    padding: $baseline/2;
     cursor: pointer;
     &:focus,
     &:hover {
       @include transition(all, $transition);
-      background-color: $blk-010;
-      outline: none;
+      border-color: $dark-gray;
+      color: $off-black;
+      position: relative;
+    }
+    &:active {
+      color: $blue;
     }
   }
 
   @mixin accordion-content() {
     display: none;
+    padding: $baseline/2 0;
+    color: $off-black;
   }
```

Comment:
See comment above about the structure of these accordion divs - if that needs to be addressed we'll want to add corresponding x-padding here to match that of the accordion-title

Reply:
This one should be good